src/widgets/ot_color_picker.eliomi

```diff
@@ -23,9 +23,16 @@
 
 (** This module implements a color picker.  *)
 
-val hsv_to_rgb : int -> float -> float -> (float * float * float)
+val hsv_to_rgb : int -> float -> float -> float * float * float
 (** [hsv_to_rgb h s v] converts HS(V/L) colors to RGB. *)
 
-val make : ?a: [< Html_types.div_attrib > `Class ] Eliom_content.Html.attrib list ->
-           [> `Div ] Eliom_content.Html.D.elt * (int * float * float) Eliom_shared.React.S.t
-(** [make ()] produces a color picker. *)
+val make :
+  ?a:[< Html_types.div_attrib > `Class ] Eliom_content.Html.attrib list ->
+  ?update:(int * float * float) React.E.t Eliom_client_value.t ->
+  unit ->
+  [> `Div ] Eliom_content.Html.D.elt
+  * (int * float * float) Eliom_shared.React.S.t
```

Comment:
could maybe do with a default value too?

Reply:
If you mean default value as for the signal, since a react event can be transformed into a react signal you don't need to add a default signal value as we did in the previous commits.

Reply:
No I mean a default value for the color. There have defaults values elsewhere.
There´s no real convention for its name but it´s there:

https://github.com/ocsigen/ocsigen-toolkit/blob/4b054847c767667ad88c13a97338f8fcc546fb71/src/widgets/ot_carousel.eliomi#L111
https://github.com/ocsigen/ocsigen-toolkit/blob/4b054847c767667ad88c13a97338f8fcc546fb71/src/widgets/ot_drawer.eliomi#L47
https://github.com/ocsigen/ocsigen-toolkit/blob/4b054847c767667ad88c13a97338f8fcc546fb71/src/widgets/ot_time_picker.eliomi#L50-L51
https://github.com/ocsigen/ocsigen-toolkit/blob/4b054847c767667ad88c13a97338f8fcc546fb71/src/widgets/ot_toggle.eliomi#L31

Reply:
I see, thanks for the review, a new commit will be pushed in a few minutes ! :)